<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1195</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
border-image:url(:/picture/back.png);
QPalette pal = this-&gt;palette();
 QImage bg(&quot;:/picture/back.png&quot;);
  bg = bg.scaled(this-&gt;width(),
                 this-&gt;height(),
                 Qt::IgnoreAspectRatio);
   pal.setBrush(QPalette::Background,QBrush(bg));
   this-&gt;setPalette(pal);

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget_musicList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(50,50,50,50);
color:rgb(255,255,255);
</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_musicHistory">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>200</y>
      <width>371</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(50,50,50,50);
color:rgb(255,255,255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_musicName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label{
color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string>歌名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_musicTime">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
#label_2{
color: rgb(170, 255, 0);
}
</string>
    </property>
    <property name="text">
     <string>歌曲时长</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_time">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_priMusic">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>590</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resourse.qrc">
      <normaloff>:/picture/previous.png</normaloff>:/picture/previous.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_startAndEnd">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>590</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resourse.qrc">
      <normaloff>:/picture/play.png</normaloff>:/picture/play.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_nextMusic">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>590</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resourse.qrc">
      <normaloff>:/picture/next.png</normaloff>:/picture/next.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_sound">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>530</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_musicSound">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>540</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>音量</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_c">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>600</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>列表循环</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>随机播放</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>单曲循环</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_chooseMusicList">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>520</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择歌单</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_changeUser">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>590</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>切换账号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_user">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>当前用户:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_history">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>140</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>历史记录</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1195</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resourse.qrc"/>
 </resources>
 <connections/>
</ui>
